metricbeat/docs/running-on-kubernetes.asciidoc

```diff
@@ -169,6 +169,29 @@ oc patch namespace kube-system -p \
 This command sets the node selector for the project to an empty string. If you
 don't run this command, the default node selector will skip master nodes.
 
+[float]
+==== Load {kib} dashboards
```

Comment:
Same comments as I made for CloudFoundry apply here: too much duplicated text.

Another consideration is that k8s users won't have the beats package downloaded to their local machine, so they'll need access to those steps.